lib/selenium-webdriver/extendedWebDriver.js

```diff
@@ -0,0 +1,42 @@
+// Used to provide better protractor documentation for methods given by
+// `webdriver-js-extender`
+
+/**
+ * @fileoverview Extra methods provided by webdriver-js-extender
+ */
+
+goog.provide('webdriver');
```

Comment:
Yeah, if I don't include it then `getNetworkConnection` is stripped out for some reason.